setup.py

```diff
@@ -42,7 +42,14 @@
 
 setup(
     name="baseplate",
+    description="A library to build services on",
+    long_description=open("README.rst").read(),
+    author="Neil Williams",
```

Comment:
otherwise, yeah, we could put your name here, or maybe putting `Albert Einstein` would lend us some cred